apps/storybook/stories/Namekit/DisplayedPrice.stories.tsx

```diff
@@ -0,0 +1,365 @@
+  },
+};
+
+export const LargePriceDisplaySize: Story = {
+  args: {
+    price: {
+      currency: Currency.Usd,
+      value: 198900n,
+    },
+    altPrice: {
+      currency: Currency.Eth,
+      value: 1000000000000000000n,
+    },
+    priceTextDisplaySize: PriceDisplaySize.Large,
+  },
+};
+export const MediumPriceDisplaySize: Story = {
+  args: {
+    price: {
+      currency: Currency.Usd,
+      value: 198900n,
+    },
+    altPrice: {
+      currency: Currency.Eth,
+      value: 1000000000000000000n,
+    },
+    priceTextDisplaySize: PriceDisplaySize.Medium,
+  },
+};
+export const SmallPriceDisplaySize: Story = {
+  args: {
+    price: {
+      currency: Currency.Usd,
+      value: 198900n,
+    },
+    altPrice: {
+      currency: Currency.Eth,
+      value: 1000000000000000000n,
+    },
+    priceTextDisplaySize: PriceDisplaySize.Small,
+  },
+};
+export const MicroPriceDisplaySize: Story = {
+  args: {
+    price: {
+      currency: Currency.Usd,
+      value: 198900n,
+    },
+    altPrice: {
+      currency: Currency.Eth,
+      value: 1000000000000000000n,
+    },
+    priceTextDisplaySize: PriceDisplaySize.Micro,
+  },
+};
+export const DisplaySymbologyAtLeft: Story = {
+  args: {
+    price: {
+      currency: Currency.Usd,
+      value: 198900n,
+    },
+    altPrice: {
+      currency: Currency.Eth,
+      value: 1000000000000000000n,
+    },
+    symbolPosition: CurrencySymbolPosition.Left,
+  },
+};
+export const DisplaySymbologyAtRight: Story = {
+  args: {
+    price: {
+      currency: Currency.Usd,
+      value: 198900n,
+    },
+    altPrice: {
+      currency: Currency.Eth,
+      value: 1000000000000000000n,
+    },
+    symbolPosition: CurrencySymbolPosition.Right,
+  },
+};
+export const DisplaySymbolSymbologyForETH: Story = {
+  args: {
+    price: {
+      currency: Currency.Eth,
+      value: 1000000000000000000n,
+    },
+    altPrice: {
+      currency: Currency.Usd,
+      value: 198900n,
+    },
+    currencySymbology: CurrencySymbology.Symbol,
+  },
+};
+export const DisplayAcronymSymbologyForETH: Story = {
+  args: {
+    price: {
+      currency: Currency.Eth,
+      value: 1000000000000000000n,
+    },
+    altPrice: {
+      currency: Currency.Usd,
+      value: 198900n,
+    },
+    currencySymbology: CurrencySymbology.Acronym,
+  },
+};
+export const DisplaySymbolSymbologyForUSD: Story = {
+  args: {
+    price: {
+      currency: Currency.Usd,
+      value: 198900n,
+    },
+    altPrice: {
+      currency: Currency.Eth,
+      value: 1000000000000000000n,
+    },
+    currencySymbology: CurrencySymbology.Symbol,
+  },
+};
+export const DisplayAcronymSymbologyForUSD: Story = {
+  args: {
+    price: {
+      currency: Currency.Usd,
+      value: 198900n,
+    },
+    altPrice: {
+      currency: Currency.Eth,
+      value: 1000000000000000000n,
+    },
+    currencySymbology: CurrencySymbology.Acronym,
+  },
+};
+export const LargeAltPriceDisplaySize: Story = {
```

Comment:
The vertical alignment for all the Alt Price stories is broken.

![CleanShot 2024-07-01 at 03 24 24](https://github.com/namehash/nameguard/assets/126201998/7184f955-2b9a-4161-bebe-acca6141853b)

Reply:
@lightwalker-eth it seems to me we need to review the use cases we want to support related to price size:

The spacing different is due to the line-height difference of these small & big texts, respectively -- IDK if someone would want to display alt. price such big and main price such small -- maybe it is a good idea to understand which pairs of sizes we want to support and make sure these are correctly displayed instead of creating multiple conditionals, which would be needed, in order to align these frankstein use cases?

e.g. we give support for prices in size X and alt. prices in size X, X - 1 and X - 2, meaning --- we give support for price in Large size! With this, be sure alt. price Large, Medium and Small size are also supported! 

Besides from this use case where the alt. price is displayed bigger than the main price, the previous comments where you identified two sizes with same icon size motivated me to think we should do a little formatting on this component's specification --- What do you think?

Reply:
@FrancoAguzzi Open to your suggestions. I'm not a CSS guy so I don't have much advice to share.

Without thinking of CSS technical details, from an end-user perspective, I just want all the text to feel like it is sitting on the same "table" such that the vertical alignment of the text appears consistent.

Reply:
Perfect, @lightwalker-eth 

As you, all users would likely want the same!

This is another business decision to be done! Such as [this one](https://github.com/namehash/nameguard/pull/288#discussion_r1674904531)

I think using alt price bigger than main price does not make sense so I like the suggestion I gave in my previous comment 😅 @notrab could you read it as well and give as your opinion? @lightwalker-eth I also encourage you to opinate on this price + alt price displaying strategies to be open-sourced!

⛓

Reply:
@FrancoAguzzi Not sure if I understand your question here. I won't be able to help much with the CSS specifics.

If your question is to ask if we can assume a constraint where the alt price is never bigger than the primary price, sure, please proceed with that.

Please work backwards from the goal here: We need to ensure the alignment issue here is fixed. If fixing this means we need to add in a constraint somewhere else, please introduce that constraint and document it.

Reply:
Sounds perfect, @lightwalker-eth

Reply:
@FrancoAguzzi Ok. Looking forward to seeing your proposal for fixing this.